garrysmod/gamemodes/terrortown/gamemode/traitor_state.lua

```diff
@@ -1,4 +1,4 @@
-
+util.AddNetworkString("ttt_spectate")

```

Comment:
I would personally prefer to move this to init.lua.
https://github.com/garrynewman/garrysmod/blob/master/garrysmod/gamemodes/terrortown/gamemode/init.lua#L125-#L156